### ISSUE TITLE
The Web Token must have a signature at the end. The incoming token did not have a signature at the end of the token.

### ISSUE THREAD

**Issue by khanly** [kkpoon/hubot-microsoft-translator#1]
Hello.

I tested hubot-microsoft-translator after the installation and it worked.  However, when I got back later to test again, I got the following error:
-------
The Web Token must have a signature at the end. The incoming token did not have a signature at the end of the token.
-------

Please advise.

**Comment by upbeta01** [comment]
Yeah, I am also getting the same error.. even on a new install hubot instance.

`Failed to parse translate result: <html><body><h1>Argument Exception</h1><p>Method: Translate()</p><p>Parameter: </p><p>Message: The Web Token must have a signature at the end. The incoming token did not have a signature at the end of the token.</p><code></code><p>message id=0151.V2_Rest.Translate.8664F37</p></body></html>, error: TypeError: Cannot read property '_' of undefined`

**Comment by kkpoon** [comment]
Hi, thanks for the report. However I currently have no time to investigate the problem. need some help.

**Comment by ubiko** [comment]
@khanly 
@upbeta01 
@kkpoon 

I found a workaround, editing `src/microsoft-translator.coffee` and removing process.env.clientID etc and replacing them directly with your ID/SecretToken as strings. will work
